{{cookiecutter.project_slug}}/server/{{cookiecutter.project_slug}}/settings.py

```diff
@@ -329,23 +339,20 @@
         "console": {
             "level": "INFO",
             "class": "logging.StreamHandler",
+            "filters": ["fix_strings"],
             "formatter": "verbose",
         },
     },
     "loggers": {
         "django": {
-            "handlers": [
-                "console",
-            ],
-            "level": "INFO",
+            "handlers": ["console"],
+            "level": "ERROR",
         },
         # The logger name matters -- it MUST match the name of the app
         "{{ cookiecutter.project_slug }}": {
-            "handlers": [
-                "console",
-            ],
+            "handlers": ["console"],
             "level": "DEBUG",
-            "propagate": True,
+            "propagate": False,
```

Comment:
This solves the duplicate logs issue. When you propagate, the log event also goes up to the parent logger. If that logger also logs it, then you'll end up with 2 events in a row every time.